src/main/java/com/google/sps/data/EventStorage.java

```diff
@@ -19,10 +19,39 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
 
 public class EventStorage {
-  public static Event getEvent(String event_id) {
-    // TODO: Query in datastore.
+  public static Event getEvent(String eventId) {
+    Query query = new Query("Event").setFilter(new FilterPredicate(Entity.KEY_RESERVED_PROPERTY, FilterOperator.EQUAL, KeyFactory.createKey("Event", eventId)));
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity eventEntity = datastore.prepare(query).asSingleEntity();
+    if (eventEntity != null) {
+      return new Event(
+        eventId, (String) eventEntity.getProperty("gcalendar-id"),
+        (String) eventEntity.getProperty("title"),
+        (String) eventEntity.getProperty("description"),
+        (String) eventEntity.getProperty("category"),
+        (ArrayList) eventEntity.getProperty("tags"),
+        new Gson().fromJson((String) eventEntity.getProperty("date-time-range"), DateTimeRange.class),
+        (Long) eventEntity.getProperty("duration"),
+        (String) eventEntity.getProperty("location"),
+        (ArrayList) eventEntity.getProperty("links"),
+        new Gson().fromJson((String) eventEntity.getProperty("fields"), Map.class),
+        (String) eventEntity.getProperty("owner"),
+        (ArrayList) eventEntity.getProperty("invited-users"),
+        (ArrayList) eventEntity.getProperty("joined-users"),
+        (ArrayList) eventEntity.getProperty("declined-users")
+      );
```

Comment:
Maybe sync with Rebeka, but this conversion Entity -> Event will occur on many places. Make a function in the Event class for the conversion
`Event.fromDatastoreEntity(entity)` which also does some try/catch things and return null if it is not successful.